css/properties.json

```diff
@@ -703,24 +707,24 @@
     "groups": [
       "WebKit Extensions"
     ],
-    "initial": "<code>rgb(0, 0, 0)</code>",
+    "initial": "<code>currentColor</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "'color'",
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
   "-webkit-text-stroke-width": {
     "syntax": "&lt;length&gt;",
     "media": "visual",
     "inherited": true,
-    "animationType": "discrete",
+    "animationType": "no",
```

Comment:
Thank you for the explanation. I shall file an issue to Web Compatibility Specification for updating this.